main.js

```diff
@@ -0,0 +1,46 @@
+const collection = document.querySelector('.collection');
+const titleInput = document.querySelector('.title-input');
+const authorInput = document.querySelector('.author-input');
+const addBtn = document.querySelector('.add-btn');
+let books = [];
+
+function removefromDOM() {
+  const removeBtn = document.querySelectorAll('.remove-btn');
+  removeBtn.forEach((elem, index) => {
+    elem.addEventListener('click', () => {
+      elem.parentNode.remove();
+      books.splice(index, 1);
+      localStorage.setItem('books', JSON.stringify(books));
+    });
+  });
```

Comment:
[ Optional ]
I can see a book can be deleted by this logic, but it would be nice for you to have used the filter method as it is elegant and more functional programming in nature.

**Example Code**
```mdx
const removeFromDOM = (bookID) => {
  // bookID can be any other property, e.g. title or author, in as much it is unique
 // You could use destructuring in the filter method to get the id from a book that is being looped over.
  books = books.filter(({ id }) => bookID !== id); // This assumes that `bookID` parameter is the book id
};
```

Reply:
Thank you @AmaduKamara. I will set it up using the filter method.